### PR TITLE
Move experiment table context menus to right-start

### PR DESCRIPTION
**Pull request by mattseddon**
Addresses the following point from https://github.com/iterative/vscode-dvc/issues/2241

>  - [ ] All context menus need to appear on the right side of the target point.
The context menu is usually placed on the right/bottom side of the mouse click unless it is close to the right age of the screen, then it will appear on the left side. However, in VSCode it seems like always on the right side, but we can do it correctly I think.

### Demo

https://user-images.githubusercontent.com/37993418/189559146-9b12a998-6009-459b-8b5d-1436256104e7.mov

### CODE REVIEW

File: webview/src/shared/components/contextMenu/ContextMenu.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const ContextMenu: React.FC<ContextMenuProps> = ({
   <Tooltip
     trigger={trigger}
     delay={[100, 200]}
-    placement={'bottom'}
+    placement={'right-start'}
```
Comment (mattseddon) [inline webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: All possible placements are [here](https://atomiks.github.io/tippyjs/#placements).

Comment (shcheklein) [reply webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: should we do right-end?

Comment (mattseddon) [reply webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: You would think so but:


https://user-images.githubusercontent.com/37993418/189559868-9a5be8ec-d7f0-419e-9e18-cb93f2f394e5.mov

**Review by julieg18 — APPROVED** [review]

**Review by sroy3 — APPROVED** [review]

**Review by wolmir — APPROVED** [review]

File: webview/src/shared/components/contextMenu/ContextMenu.tsx (hunk 1 above)
Comment (maxagin) [reply webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: Normally it's bottom-end. No?

Comment (mattseddon) [reply webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: bottom-end:


https://user-images.githubusercontent.com/37993418/189771989-477e192d-e26c-4425-b143-8cea497ce1bd.mov

Comment (maxagin) [reply webview/src/shared/components/contextMenu/ContextMenu.tsx:59]: Maybe I was not clear enough, sorry. This is an example from Figma 

Screen Shot 2022-09-12 at 7 26 32 PM

**Pull request merged**